swift/looker/Tests/lookerTests/methodsTests.swift

```diff
@@ -26,16 +26,16 @@ class methodsTests: XCTestCase {
     
     func simpleQuery() -> WriteQuery {
         return WriteQuery(
-            model: "system__activity",
-            view: "dashboard",
+            "system__activity",
+            "dashboard",
             fields: ["dashboard.id", "dashboard.title", "dashboard.count"],
             limit: "100")
```

Comment:
this is how the constructor call changes

Reply:
Hmm, so it changes the call sites? Does this change just expose the constructors outside of the SDK package? If so, then I guess the change would be isolated to just sdk-codegen, so maybe that's okay.

Reply:
Both patterns are now supported. win/win.